README.md

```diff
@@ -0,0 +1,64 @@
+# Breadcrumb
+Breadcrumb is a canonical component that any VTEX app can use.
+
+To use it in your code you should first add it as a dependencie into your app's ```manifest.json```, like:
+
+```json
+"dependencies": {
+    "vtex.breadcrumb": "0.x",
+    "vtex.category-menu": "0.x"
+}
+```
+
+## Usage
+You can use it in your code through an extension point in your app.
+
+```javascript
+    <ExtensionPoint
```

Comment:
Show direct usage of the component, not the `ExtensionPoint`

Reply:
But the component will never be used directly, whats the point of that?

Reply:
The point is to document the usage of the _own_ component, so in the future, if we remove it from the extension point and use it directly or somebody else need to use it in another app but don't want to put in an extension point, we'll have the proper documentation for that

Reply:
Even though I agree with you in parts, I strongly believe that the document should reflect the real world usage, not a theoretical version of it. Moreover, this document is not immutable, so if we change the implementation of the component or just the usage of it, this changes should be reflected in this document. Therefore, I ask you to reconsider your change request.